dashboard/app/models/programming_class.rb

```diff
@@ -110,6 +110,32 @@ def summarize_for_edit
     }
   end
 
+  def summarize_for_show
+    {
+      id: id,
+      key: key,
+      name: name,
+      content: content,
+      examples: parsed_examples,
+      fields: parsed_fields,
+      tips: tips,
+      syntax: syntax,
+      external_documentation: external_documentation,
```

Comment:
`external_documentation` is snake case and  `categoryKey` is camel case, it'd be great to have consistency

Reply:
Ha, I found this issue working on something else. It should be camel case and I think it's not working in a number of places because of this. I'll fix it here, and in the above function